src/main/java/org/dimdev/dimdoors/block/DimensionalPortalBlock.java

```diff
@@ -114,7 +114,7 @@ public void onBlockAdded(BlockState state, World world, BlockPos pos, BlockState
 	}
 
 	private static void portalTick(World world, BlockPos pos, BlockState state, EntranceRiftBlockEntity e) {
-		if (world.isClient || e.getDestination() != null) {
+		if (world.isClient || e.getDestination().getLocation() != null) {
```

Comment:
I believe the destination may be null. Should probably test for both the destination being null and its location being null.

Reply:
The destination is initialized to NoneTarget and I didn't find a place where it is set to null.
https://github.com/DimensionalDevelopment/DimDoors/blob/5bda2b8017cbc558cbeb20989f414149a001b384/src/main/java/org/dimdev/dimdoors/block/entity/RiftData.java#L9-L10

Reply:
> 
> 
> The destination is initialized to NoneTarget and I didn't find a place where it is set to null.
> 
> https://github.com/DimensionalDevelopment/DimDoors/blob/5bda2b8017cbc558cbeb20989f414149a001b384/src/main/java/org/dimdev/dimdoors/block/entity/RiftData.java#L9-L10

Heheheh. I was very meticulous when doing the NoneTarget conversion. ^^